### ISSUE TITLE
Touchpoints launcher on static site is not contained by a landmark

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4038]
## Area or Component
Static site

## Quick Summary
The touchpoints button launcher is not contained by a landmark (main, header, or footer).

## Accessibility Violation
All page content should be contained by landmarks: https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome

## Action Requested
Ensure the Touchpoints launcher is contained by a landmark. It will probably need to go in `<main>` or `<footer>` (though we already have a footer on the site, so be careful not to duplicate one)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/16udNpwdJ8sltAULnmSIGgkWVQsfuFvcoymP0-s_QwfE/edit#gid=1510420308&range=8:8

**Comment by BobanL** [comment]
@emmastephenson when I'm looking at the static site it seems that the touchpoint is in the footer or is there something else I am missing?
<img width="493" alt="image" src="https://user-images.githubusercontent.com/10108172/186460136-1d6a758d-218b-4ffa-bb69-f579e70a6b4c.png">

**Comment by emmastephenson** [comment]
@BobanL Hmmm 🤔 I think that @fzhao99 fixed this but maybe just closed the wrong issue? That's what I'm seeing with #4002. So this can be closed, but the main site one should still be fixed.
